src/meta/src/manager/catalog/database.rs

```diff
@@ -15,10 +15,11 @@
 use std::collections::hash_map::Entry;
 use std::collections::{HashMap, HashSet};
 
+use itertools::Itertools;
 use risingwave_pb::catalog::{Database, Index, Schema, Sink, Source, Table};
 
 use super::{DatabaseId, RelationId, SchemaId, SinkId, SourceId};
```

Comment:
Need to rename `RelationId` and `RelationKey` ?

Reply:
why? I think RelationKey is actually another thing. `type RelationKey = (DatabaseId, SchemaId, String);`

Reply:
Both `RelationId` and `RelationKey` are related to `Relation`. But we rename `Relation` to `StreamingJob`

Reply:
The original `Relation` definition is a little bit confusing, and it should be something like `StreamJob`. Because it only works for creating stream jobs on CNs. But the `RelationId` and `RelationKey` are works for catalog management. Actually they are different concepts and there is no connection between them.